src/board/epd_board_v7.c

```diff
@@ -248,6 +248,53 @@ static void epd_board_poweron(epd_ctrl_state_t* state) {
     }
 }
 
+static void epd_board_measure_vcom(epd_ctrl_state_t* state) {
```

Comment:
Since this is the same as on v6, can we refactor it out into a reusable function so we don't duplicate it?

Reply:
Sorry Valentin but I'm not really going to this for two duplicate functions since needs refactoring many things and does NOT make any sense for a funcion that only 1% of the epdiy users will ever use.
So you can accept as it is, or you can propose a different refactoring, but I'm not willing to do it just for this. For more info: Doing this would imply doing a common_v6 (While there is already a common for the boards but from v2 to v5) so you need to really add more files. And it's not prepare to do that since it will imply that many variables + includes have to be moved there (Like #include "pca9555.h" #include "tps65185.h" etc)
And does not really make it just to avoid a single duplicate function.